srcs/juloo.keyboard2/Config.java

```diff
@@ -147,6 +150,10 @@ public void refresh(Resources res)
     keyboardOpacity = _prefs.getInt("keyboard_opacity", 100) * 255 / 100;
     keyOpacity = _prefs.getInt("key_opacity", 100) * 255 / 100;
     keyActivatedOpacity = _prefs.getInt("key_activated_opacity", 100) * 255 / 100;
+  	// keyboard border settings
+		borderConfig = _prefs.getBoolean("border_config", false);
+		borderRadius = _prefs.getInt("border_radius", 0);
```

Comment:
The preference contain numbers in the `dp` unit but `borderRadius` is in `px` unit. It must be converted like this:
```suggestion
		borderRadius = get_dip_pref(dm, "border_radius", 0);
```
Same for borderLineSize.